beerus_cli/Cargo.toml

```diff
@@ -18,3 +18,4 @@ env_logger = "0.10.0"
 starknet = { git = "https://github.com/xJonathanLEI/starknet-rs" }
 serde_json = "1.0.91"
 base64 = "0.20.0"
+serde = "1.0.150"
```

Comment:
Do we need it ?

Reply:
Yes, to deserialize a TransactionObject from a json input